pyproject.toml

```diff
@@ -24,8 +24,7 @@ dependencies  = [
 [project.optional-dependencies]
 test = [
     "scipy",
-    "matplotlib",
-    "pytest"
+    "matplotlib"
```

Comment:
why?

Reply:
Since `pytest` is already listed as a core dependency, it is already installed after running `pip install -e .`
Unless I am misunderstanding why it is listed as an optional dependency.